integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -187,6 +188,13 @@ public Long load(String groupName) {
     mIsUserGroupTranslation = conf.getBoolean(PropertyKey.FUSE_USER_GROUP_TRANSLATION_ENABLED);
     mMaxUmountWaitTime = (int) conf.getMs(PropertyKey.FUSE_UMOUNT_TIMEOUT);
     mAuthPolicy = AuthPolicyFactory.create(mFileSystem, conf, this);
+    if (opts.isDebug()) {
+      try {
+        LogUtils.setLogLevel(this.getClass().getCanonicalName(), "DEBUG");
```

Comment:
Can use `org.slf4j.event.Level.DEBUG.toString()` ?
Try to avoid hardcoding strings